src/Momento.Sdk.Incubating/Momento.Sdk.Incubating.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 25.0.1704.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Momento.Sdk.Incubating", "Momento.Sdk.Incubating.csproj", "{5F3FFF72-091E-4D3A-9804-D18F2CB246A8}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{5F3FFF72-091E-4D3A-9804-D18F2CB246A8}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{5F3FFF72-091E-4D3A-9804-D18F2CB246A8}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{5F3FFF72-091E-4D3A-9804-D18F2CB246A8}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{5F3FFF72-091E-4D3A-9804-D18F2CB246A8}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {D5A06D91-DEFE-4D6D-920F-5A11BE7D0E40}
+	EndGlobalSection
+EndGlobal

```

Comment:
This shouldn't be here? We have a solution file already at the top level of the project.